### DISCUSSION TITLE
Shard replication to specific worker

### DISCUSSION THREAD

**Discussion by huseynsnmz** [citusdata/citus#4623] [Q&A] [answered]
Hello,

I want to use `citus.shard_replication_factor = 2` in case of worker failure. I want to replicate the shard one of the workers that I choose because some of the workers on different disks. Is it possible?

**Comment by marcocitus** [comment, marked as answer]
Not in a currently released version, but when using the master branch (release is < 1 month away), this can be done using:
```sql
select citus_copy_shard_placement(102300, 'alive-node', 5432, 'new-node', 5432, do_repair := false);
```

**Reply by huseynsnmz** [reply]
Awesome. Thank you.
